frontend/src/app/esri-map/esri-map.component.ts

```diff
@@ -59,10 +58,32 @@ export class EsriMapComponent implements AfterViewInit {
         });
 
         function checkForest (feature) {
+          let fwfRoute;
+          let fwf;
           const mbsForest = 'Mt. Baker-Snoqualmie National Forest';
           const common = feature.graphic.attributes.COMMONNAME;
           const route = document.location.origin;
-          if (common === mbsForest) {
+          const fuelwoodForest = [
+            {fname: 'Idaho Panhandle National Forests', fshort: 'ipnf'},
+            {fname: 'Flathead National Forest', fshort: 'flathead'},
+            {fname: 'Chattahoochee-Oconee National Forest', fshort: 'conf'},
+            {fname: 'George Washington & Jefferson National Forest', fshort: 'gwj'},
+            {fname: 'Shoshone National Forest', fshort: 'shoshone'},
+            {fname: 'Hiawatha National Forest', fshort: 'hiawatha'},
+            {fname: 'Grand Mesa Uncompahgre & Gunnison National Forest', fshort: 'gmug'},
+          ];
```

Comment:
Can this const array be moved outside of any functions, so that it isn't created on each function call? It's a small performance thing.